File: src/main/java/xyz/aeolia/ashutils/task/MessageLaterTask.java
package xyz.aeolia.ashutils.task;

import org.bukkit.command.CommandSender;
import org.bukkit.scheduler.BukkitRunnable;
import xyz.aeolia.ashutils.sender.MessageSender;

public class MessageLaterTask extends BukkitRunnable {
  CommandSender sender;
  String message;

  public MessageLaterTask(CommandSender sender, String message) {
    this.sender = sender;
    this.message = message;
  }

  @Override
  public void run() {
    MessageSender.sendMessage(sender, message);
  }
}

File: src/main/java/xyz/aeolia/ashutils/task/MessageLaterTask.kt
package xyz.aeolia.ashutils.task

import jdk.internal.joptsimple.internal.Messages.message
import org.bukkit.command.CommandSender
import org.bukkit.scheduler.BukkitRunnable
import xyz.aeolia.ashutils.sender.MessageSender

class MessageLaterTask(val recipient: CommandSender, val message: String) : BukkitRunnable() {
  override fun run() {
    MessageSender.sendMessage(recipient, message, true)
  }
}